documentation/markup.rst

```diff
@@ -30,11 +30,21 @@ external links          ```Link text <https://example.com>`_``      :ref:`hyperl
 roles w/ custom text    ``:role:`custom text <target>```            :ref:`roles`
 roles w/ only last part ``:role:`~hidden.hidden.visible```          :ref:`roles`
 roles w/o link          ``:role:`!target```                         :ref:`roles`
+roles w/o link, short   ``:role:`!visible``` (Note 1)               :ref:`roles`
 issues                  ``:gh:`ID```, ``:issue:`ID```               :ref:`roles`
 CPython source          ``:source:`PATH```                          :ref:`roles`
 comments                ``.. a comment``                            :ref:`comments`
 ======================= =========================================== ====================
 
+Notes:
+
+(1)
+   For an only-last-part reference with a suppressed link,
+   ``:role:`~!hidden.visible``` makes more semantic sense, but it causes
```

Comment:
semantic (adj): related to meaning, or to the study of meaning

Therefore semantic sense or semantic meaning is redundant 🙂 

```suggestion
   ``:role:`~!hidden.visible``` makes more sense, but it causes
```